packages/console/src/pages/EnterpriseSso/Guide/SamlMetadataForm/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { conditional, pick } from '@silverhand/essentials';
+import { useState } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import FormField from '@/ds-components/FormField';
+import TextInput from '@/ds-components/TextInput';
+import Textarea from '@/ds-components/Textarea';
+
+import { type SamlGuideFormType } from '../../types.js';
+
+import SwitchFormatButton, { FormFormat } from './SwitchFormatButton';
+import * as styles from './index.module.scss';
+
+/**
+ * Since we need to reset some of the form fields when we switch the form format
+ * for SAML configuration form, here we define this object for convenience.
+ */
+const completeResetObject: Omit<SamlGuideFormType, 'attributeMapping'> = {
+  metadata: undefined,
+  metadataUrl: undefined,
+  signInEndpoint: undefined,
+  entityId: undefined,
+  x509Certificate: undefined,
+};
+
+type Props = {
+  formFormat: FormFormat;
+};
+
+function GenerateFormByFormat({ formFormat }: Props) {
```

Comment:
That does not look like a component name.

Reply:
Updated.